sdk/appconfiguration/tests.yml

```diff
@@ -22,8 +22,6 @@ extends:
         China:
           SubscriptionConfiguration: $(sub-config-cn-test-resources)
           Location: chinanorth3
-      MatrixReplace:
-        - TestSamples=.*/true
```

Comment:
If it's just one sample failing, you can disable it here: https://github.com/Azure/azure-sdk-for-python/blob/main/scripts/devops_tasks/test_run_samples.py#L81